src/view/drag-handle/drag-handle.jsx

```diff
@@ -71,20 +80,53 @@ export default class DragHandle extends Component<Props> {
     this.canLift = context[canLiftContextKey];
   }
 
-  componentWillUnmount() {
-    this.sensors.forEach((sensor: Sensor) => {
-      // kill the current drag and fire a cancel event if
-      const wasDragging = sensor.isDragging();
+  componentDidMount() {
+    const draggableRef: ?HTMLElement = this.props.getDraggableRef();
 
-      sensor.unmount();
-      // cancel if drag was occurring
-      if (wasDragging) {
-        this.props.callbacks.onCancel();
-      }
-    });
+    // storing a reference for later
+    this.lastDraggableRef = draggableRef;
+
+    if (!draggableRef) {
+      console.error('Cannot get draggable ref from drag handle');
+      return;
+    }
+
+    // drag handle ref will not be available when not enabled
+    if (!this.props.isEnabled) {
```

Comment:
Unrelated to this PR but any reason why this prop isn't called `isDisabled` instead? It wasn't immediately obvious to me what this prop meant exactly. Changing this to `isDisabled` also aligns with the draggable's disabled prop.

Reply:
I generally favour positive props for internal usage. As you mentioned, not part of this PR - but perhaps work thinking more about